### PR TITLE
Added code for xclLogMsg to AWS shim & awssak

### PR DESCRIPTION
**Pull request by sarab96**
Added code for xclLogMsg to AWS shim & awssak
This is for 2019.1 release. Merging to master branch only.
@sonals  Please check
@rradjabi  Please review changes to CMakeList as well

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

File: src/runtime_src/driver/xclng/tools/awssak/CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +32,8 @@ else()
     xrt_awsstatic
     pthread
     rt
+    boost_filesystem
```
Comment (rradjabi) [inline src/runtime_src/driver/xclng/tools/awssak/CMakeLists.txt]: Will including boost for Amazon be an issue?

Comment (sarab96) [reply src/runtime_src/driver/xclng/tools/awssak/CMakeLists.txt]: We now have boost as part of XRT and it was always required for OCL library. So it is okay to include boost for AWS also.

**Comment by sonals** [comment]
Please close this PR. We plan to move to common XRT library so we should not need to duplicate this.

**Comment by sarab96** [comment]
Closing based on Sonal's comment.

**Pull request closed without merging**